packages/unit-testing/README.md

```diff
@@ -11,7 +11,7 @@ The following software is required to use SuiteCloud Unit Testing package:
 ## `jest.config.js` File Examples
```

Comment:
In Prerequisites section of this file it says:
The following software is required to use SuiteCloud Unit Testing package: Node.js, Jest, _tests_ folder and jest.config.js file.
I think jest.config.js file doesn't belong to this section - it is configuration and not software
 _tests_ folder is not required but is the default location for placing test files, I think it can be overriten in config so I would only add a suggestion to create _tests_ folder according to Jest conventions

Reply:
As I understood, you need the **jest.config.js** file in order to configure your jest config, so that's why I added it here. However, since the file and the folder are not software, I can create a second list with required configuration and add those two there. What do you think?

Also, the **__tests__** folder is added when running `project:create` and adding unit testing, so I think it's good to add this as our config structure.